lib/PostMessageTransport.js

```diff
@@ -57,8 +79,20 @@ define(function (require, exports, module) {
      */
     function send(idOrArray, msgStr){
         var win = _iframeRef.contentWindow;
+        var msg = JSON.parse(msgStr);
+
+        if(msg && msg.params && msg.params.expression) {
+            msg.params.expression = resolveLinks(msg.params.expression);
+        }
+
+        msgStr = JSON.stringify(msg);
 
         win.postMessage(msgStr,"*");
+
+        if(msg.method === "Page.reload" || msg.method === "Page.navigate") {
```

Comment:
If I remember correctly, Brackets handles that internally. So, it doesn't send a `Page.reload` message unless it is an HTML page